### PR TITLE
Universal/docs/post processing docs

### PR DESCRIPTION
**Pull request by reallyfancy**
---
### Purpose of this PR
Added docs for the following post-processing effects: Bloom, Chromatic Aberration, Depth of Field, Tonemapping, Vignette. Also updated table of contents, and fixed a couple of typos/old names.

Changes reviewed by Thomas Hourdel.

### CODE REVIEW

**Comment by phi-lira** [comment]
LGTM. Adding @Chman and @rainsing for 2D and Post-processing coverage.

File: com.unity.render-pipelines.universal/Documentation~/2d-index.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
\ No newline at end of file
+You can choose from several different light types with the **Light 2D** component. The light types currently available in the package are:
+
+- [Freeform](LightTypes.html#freeform) 
+- [Sprite](LightTypes.html#sprite)
+- [Parametric](LightTypes.html#parametric)
+- [Point](LightTypes.html#point)
+- [Global](LightTypes.html#global)
+
+![](images\image_1.png)
```
Comment (Sam-Unity) [inline com.unity.render-pipelines.universal/Documentation~/2d-index.md]: Broken link, change to ``

**Comment by phi-lira** [comment]
@reallyfancy what's missing for this one?

**Comment by reallyfancy** [comment]
> @reallyfancy what's missing for this one?

Nothing as of now! Fixed issue raised by Sam, which was just a broken image link.

**Pull request merged**